packages/dev-env/src/ozone-service-profile.ts

```diff
@@ -0,0 +1,412 @@
+import { TestPds } from './pds'
+import { AtpAgent } from '@atproto/api'
+import { Secp256k1Keypair } from '@atproto/crypto'
+
+export class OzoneServiceProfile {
+  did: string
+  key: Secp256k1Keypair
+  thirdPartyPdsClient: AtpAgent
+
+  modUserDetails = {
+    email: 'mod-authority@test.com',
+    handle: 'mod-authority.test',
+    password: 'hunter2',
+  }
+
+  public constructor(public thirdPartyPds: TestPds) {
+    this.thirdPartyPdsClient = this.thirdPartyPds.getClient()
+  }
+
+  async createDidAndKey() {
+    const modUser =
+      await this.thirdPartyPdsClient.api.com.atproto.server.createAccount(
+        this.modUserDetails,
+      )
+    await this.thirdPartyPdsClient.login({
+      identifier: this.modUserDetails.handle,
+      password: this.modUserDetails.password,
+    })
+
+    this.did = modUser.data.did
+    this.key = await Secp256k1Keypair.create({ exportable: true })
+    return { did: this.did, key: this.key }
+  }
+
+  async createServiceDetails(pds: TestPds, ozoneUrl: string) {
+    const pdsClient = pds.getClient()
+    const describeRes = await pdsClient.api.com.atproto.server.describeServer()
```

Comment:
This mostly mimics the way migration is done in [account-migrations.test.ts](https://github.com/bluesky-social/atproto/blob/main/packages/pds/tests/account-migration.test.ts#L78)